toolkit/tools/pkg/specreaderutils/specreaderutil.go

```diff
@@ -0,0 +1,706 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package specreaderutils
```

Comment:
Optional Opinion (aka `bikeshedding`):

IMO, idiomatic go refrains from the use of the name `util`. IMO the package should   be `specreader` and the resulting binary would leverage this :)

Reference: https://go.dev/blog/package-names

Reply:
Thanks for sharing the reference. Excellent observation. We have a `specreader` folder with the `specreader.go` app, so I am not sure about renaming `specreaderutils` to `specreader`. Agree about dropping the `utils`. I'll target renaming as we do more refactoring.

Reply:
@neha170, how would having a `specreader` directory conflict with havig a `specreader` package in `pkg` ?